Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,377 @@
+            | |   / /        ||
+            | |  / /         | ----..
+            | | / /          /      \\
+            | |/ /           |   `/, |
+            | | /             \\ -   /
+            | |               ||````
+            | |               ||
+            | |               ||
+            | |               ||
+            | |
+            | |
+            | |
+            | |
+            | |
+            | |
+            =======================================
+
+                You have 5 guesses remaining.
+
+            """
+            )
+            print(showWrongAttempts)
+        case 4:
+            print(
+                """
+            =====================
+            | |    / /       ||
+            | |   / /        ||
+            | |  / /         | ----..
+            | | / /          /      \\
+            | |/ /           |   `/, |
+            | | /             \\ -   /
+            | |              /||````
+            | |             / ||
+            | |            /  ||
+            | |            )  ||
+            | |
+            | |
+            | |
+            | |
+            | |
+            | |
+            =======================================
+
+                You have 4 guesses remaining.
+
+            """
+            )
+            print(showWrongAttempts)
+        case 3:
+            print(
+            """
+            =====================
+            | |    / /       ||
+            | |   / /        ||
+            | |  / /         | ----..
+            | | / /          /      \\
+            | |/ /           |   `/, |
+            | | /             \\ -   /
+            | |              /||\\```
+            | |             / || \\
+            | |            /  ||  \\
+            | |           )   ||   (
+            | |
+            | |
+            | |
+            | |
+            | |
+            | |
+            =======================================
+
+                You have 3 guesses remaining.
+
+            """
+          
+            )
+            print(showWrongAttempts)
+        case 2:
+            print(
+                """
+            =====================
+            | |    / /       ||
+            | |   / /        ||
+            | |  / /         | ----..
+            | | / /          /      \\
+            | |/ /           |   `/, |
+            | | /             \\ -   /
+            | |              /||\\```
+            | |             / || \\
+            | |            /  ||  \\
+            | |           )   ||   (
+            | |               /
+            | |              /
+            | |             /
+            | |          __".
+            | |
+            | |
+            =======================================
+
+                 You have 2 guesses remaining.
+
+            """
+            ) //..............................................................something is wrong with this and so forth
```

Comment:
Please remember to remove any unused or commented out code before submitting your future projects.